Task1.py

```diff
@@ -9,5 +9,10 @@ def __init__(self, *args):
         self.numbers = args
 
     def find_maximum_number(self):
-        pass
+        num1 = 0
+        for i in range(len(self)):
```

Comment:
You can't use self as a n iterable object

Reply:
Did it